build/syntax-highlight.ts

```diff
@@ -53,7 +53,7 @@ const loadAllLanguages = lazy(() => {
     "typescript",
     "uri",
     "wasm",
-    // "webidl", // block list
+    "web-idl",
```

Comment:
`webidl` is an alias of `web-idl` so both will work. 

Leaving this as `web-idl` as that's the primary key for this language def. See:
* https://prismjs.com/download.html#themes=prism&languages=web-idl
* https://github.com/PrismJS/prism/blob/59e5a3471377057de1f401ba38337aca27b80e03/components.json#L1546-L1550

Reply:
Hm, unless we already use `web-idl`, I would suggest to keep using it as `webidl`, because all other supported languages don't have a hyphen either. Wdyt?

Reply:
I'm fine with that if we can use the alias to load it. I'll have to try it out to check.